docs/index.rst

```diff
@@ -2,30 +2,138 @@
 
-.. toctree::
-   :maxdepth: 1
+Getting Started
+===============
+
+Installation
+------------
+MetSim itself is a pure Python package, but its dependencies are not. The
+easiest way to get everything installed is to use conda_. To install MetSim 
+with its recommended dependencies using the ``conda`` command line tool::
+
+    $ conda install metsim
+
+.. _conda: http://conda.io/
 
-   why-metsim
-   faq
-   examples
-   installing
-   api
-   whats-new
+We recommend using the community maintained `conda-forge 
+<https://conda-forge.github.io/>`__ channel if you need difficult\-to\-build 
+dependencies such as ``numba``::
 
-Indices and tables
-==================
+    $ conda install -c conda-forge metsim
 
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+New releases may also appear in conda-forge before being updated in the default
+channel.
+
+If you are not using ``conda`` to install MetSim you should ensure that you have
+all of the required dependencies:
+
+- Python 3.5 or 3.6
+- `xarray <http://xarray.pydata.org/>`__ (0.9.1 or later)
+- `pandas <http://pandas.pydata.org/>`__ (0.19.0 or later)
+- `numba <http://numba.pydata.org/>`__ (0.31.0 or later)
+- `netCDF4 <https://github.com/Unidata/netcdf4-python>`__ 
+- `scipy <http://scipy.org/>`__
+
+
+If you don't use conda, be sure you have the required dependencies (numba and 
+xarray) installed first. Then, install metsim with pip::
+
+    $ pip install metsim
+
+To run the test suite after installing metsim, install
+`py.test <https://pytest.org>`__ (``pip install pytest``) and run
+``py.test --verbose``.
+
+Finally, you can install MetSim directly from the source if you desire to::
+
+    $ git clone https://github.com/UW-Hydro/MetSim.git
+    $ cd MetSim
+    $ python setup.py install
+    $ py.test --verbose
+
+
+Basic Usage
+-----------
+MetSim provides a simple command line interface which is primarily operated via 
+configuration files.  For more information about the options available to be set
+in the configuration files see the :ref:`configuration` page.
+
+Once installed, MetSim can be used from the command line via:
+
+``ms /path/to/configuration [-v] [-n #]``
+
+Bracketed flags are optional; ``-v`` activates verbose mode to print messages 
+about the status of a run, and ``-n`` activates parallelism.  The number given 
+after the ``-n`` flag is the number of processes to run. A good rule of thumb is
+to use one less process than the number of processsors (or threads) that the 
+machine you are running on has.
+
+References
+==========
+
+.. [1] Bohn, T. J., B. Livneh, J. W. Oyler, S. W. Running, B. Nijssen, and D. P.
+    Lettenmaier, 2013a: Global evaluation of MTCLIM and related algorithms for 
+    forcing of ecological and hydrological models, Agr. Forest. Meteorol., 176, 
+    38-49, doi:10.1016/j.agrformet.2013.03.003.
+
+Sitemap
+=======
+.. toctree::
+    :maxdepth: 3
+ 
+    index
+    examples
+    configuration
+    api
```

Comment:
I am wondering whether it is good to put all those materials in the first page. I think it is good to put the installation section in s standalone section.